new-pipeline/style/dashboards.css

```diff
@@ -139,7 +139,7 @@
   text-align: center;
 }
 
-#dist-caption-link, #evo-caption-link {
+#dist-caption-link, #evo-caption-link, #use-counter-link {
```

Comment:
Hm, looking at your [live link](https://ecomerford.github.io/telemetry-dashboard/new-pipeline/dist.html#!cumulative=0&end_date=2018-05-24&keys=__none__!__none__!__none__&max_channel_version=nightly%252F62&measure=USE_COUNTER2_PUSHMANAGER_SUBSCRIBE_DOCUMENT&min_channel_version=null&processType=*&product=Firefox&sanitize=1&sort_keys=submissions&start_date=2018-05-07&table=0&trim=1&use_submission_date=0), this does not show up with a black text color.

Reply:
I added the css key back into the link generator in `dashboard.js`. This is how it's handled for the "More information" link, as well. I don't know exactly why, but that was the only way that I could change the color.